05.01.2024 existing install fix

smuzaffar opened the pull request.

Commit:
Added patch for #7128 bugfix

Commit:
CLing cleanup for #7128 bugfix

smuzaffar commented:
please test for CMSSW_14_0_ROOT6_X

cmsbuild commented:
A new Pull Request was created by  @smuzaffar (Malik Shahzad Muzaffar) for branch cms/master/3ef0acc.

@smuzaffar, @aandvalenzuela, @iarspider can you please review it and eventually sign? Thanks.
@rappoccio, @sextonkennedy, @antoniovilela you are the release manager for this.
cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

cmsbuild commented:
cms-bot internal usage<!-- bot cache: {"commits":{"70e0f0d53cf35cdb4d0d42ca3d82d0c2d04cd7f7":{"files":["CMakeLists.txt","cmake/modules/RootMacros.cmake","core/dictgen/src/rootcling_impl.cxx","core/metacling/src/TCling.cxx"],"squashed":false,"time":1704968809},"aeffc7467d0034c7504229ec9cf52a045316f747":{"files":["geom/gdml/src/TGDMLWrite.cxx","geom/geom/inc/TGeoTessellated.h","geom/geom/src/TGeoTessellated.cxx","geom/vecgeom/inc/LinkDef.h","geom/vecgeom/inc/TGeoVGShape.h","geom/vecgeom/src/TGeoVGShape.cxx"],"squashed":false,"time":1704956250},"feb4430b62fed5f966227a17f9b87757f4221da9":{"files":["CMakeLists.txt","cmake/modules/RootMacros.cmake","core/dictgen/src/rootcling_impl.cxx","core/metacling/src/TCling.cxx"],"squashed":false,"time":1704968809}},"emoji":{},"last_seen_sha":"70e0f0d53cf35cdb4d0d42ca3d82d0c2d04cd7f7","signatures":{}} -->

cmsbuild commented:
-1

**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-fa6318/36780/summary.html
**COMMIT**: 9f208a4dc712accd3c4e0d70d583bf44151e8373
**CMSSW**: CMSSW_14_0_ROOT6_X_2024-01-09-2300/el8_amd64_gcc12
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/root/194/36780/install.sh` to create a dev area with all the needed externals and cmssw changes.


## External Build

I found compilation error when building: 

<pre>[8393/10519] /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin/g++  -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/net/auth/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/net/auth/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/unix/../clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/unix/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/zstd/1.5.2-0530c990dd9ca27a8d966ab201ed2dfe/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/zstd/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/lz4/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/lzma/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/zip/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/thread/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/textinput/src -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/textinput/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/rint/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/meta/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/gui/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/cont/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clingutils/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clingutils/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clib/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/base/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/base/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/build/ginclude -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/net/net/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/io/io/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/io/io/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/tbb/v2021.9.0-5109e373398eaf79a8268d6a86a2f6e2/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/builtins -D__ROOFIT_NOBANNER -fdiagnostics-color=always -std=c++17 -Wno-implicit-fallthrough -Wno-noexcept-type -pipe  -Wshadow -Wall -W -Woverloaded-virtual -fsigned-char -pthread -O3 -DNDEBUG -fPIC -std=c++17 -MD -MT net/auth/CMakeFiles/RootAuth.dir/src/TAuthenticate.cxx.o -MF net/auth/CMakeFiles/RootAuth.dir/src/TAuthenticate.cxx.o.d -o net/auth/CMakeFiles/RootAuth.dir/src/TAuthenticate.cxx.o -c /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/net/auth/src/TAuthenticate.cxx
[8394/10519] /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin/g++  -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/io/sql/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/unix/../clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/unix/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/zstd/1.5.2-0530c990dd9ca27a8d966ab201ed2dfe/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/zstd/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/lz4/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/lzma/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/zip/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/thread/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/textinput/src -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/textinput/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/rint/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/meta/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/gui/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/cont/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clingutils/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clingutils/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clib/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/base/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/base/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/build/ginclude -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/io/io/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/io/io/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/tbb/v2021.9.0-5109e373398eaf79a8268d6a86a2f6e2/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/builtins -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/net/net/inc -D__ROOFIT_NOBANNER -fdiagnostics-color=always -std=c++17 -Wno-implicit-fallthrough -Wno-noexcept-type -pipe  -Wshadow -Wall -W -Woverloaded-virtual -fsigned-char -pthread -O3 -DNDEBUG -fPIC -std=c++17 -MD -MT io/sql/CMakeFiles/SQLIO.dir/src/TBufferSQL2.cxx.o -MF io/sql/CMakeFiles/SQLIO.dir/src/TBufferSQL2.cxx.o.d -o io/sql/CMakeFiles/SQLIO.dir/src/TBufferSQL2.cxx.o -c /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/io/sql/src/TBufferSQL2.cxx
[8395/10519] /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin/g++  -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/net/davix/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/unix/../clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/unix/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/zstd/1.5.2-0530c990dd9ca27a8d966ab201ed2dfe/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/zstd/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/lz4/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/lzma/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/zip/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/thread/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/textinput/src -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/textinput/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/rint/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/meta/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/gui/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/cont/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clingutils/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clingutils/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clib/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/base/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/base/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/build/ginclude -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/net/net/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/io/io/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/io/io/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/tbb/v2021.9.0-5109e373398eaf79a8268d6a86a2f6e2/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/builtins -isystem /cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02819/el8_amd64_gcc12/external/davix/0.8.1-bea8e5a05ee4ae615a84f825c78bc662/include/davix -D__ROOFIT_NOBANNER -fdiagnostics-color=always -std=c++17 -Wno-implicit-fallthrough -Wno-noexcept-type -pipe  -Wshadow -Wall -W -Woverloaded-virtual -fsigned-char -pthread -O3 -DNDEBUG -fPIC -Wno-deprecated-declarations -std=c++17 -MD -MT net/davix/CMakeFiles/RDAVIX.dir/src/TDavixFile.cxx.o -MF net/davix/CMakeFiles/RDAVIX.dir/src/TDavixFile.cxx.o.d -o net/davix/CMakeFiles/RDAVIX.dir/src/TDavixFile.cxx.o -c /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/net/davix/src/TDavixFile.cxx
[8396/10519] /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin/g++  -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/graf2d/gpadv7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/build/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/zstd/1.5.2-0530c990dd9ca27a8d966ab201ed2dfe/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/tbb/v2021.9.0-5109e373398eaf79a8268d6a86a2f6e2/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/unix/../clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/unix/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/zstd/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/lz4/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/lzma/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/zip/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/thread/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/textinput/src -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/textinput/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/rint/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/meta/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/gui/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/foundation/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/cont/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clingutils/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clingutils/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clib/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/base/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/base/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/build/ginclude -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/io/io/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/io/io/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/core/clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/root-6.31.01/builtins -D__ROOFIT_NOBANNER -fdiagnostics-color=always -std=c++17 -Wno-implicit-fallthrough -Wno-noexcept-type -pipe  -Wshadow -Wall -W -Woverloaded-virtual -fsigned-char -pthread -O3 -DNDEBUG -fPIC -std=c++17 -MD -MT graf2d/gpadv7/CMakeFiles/G__ROOTGpadv7.dir/G__ROOTGpadv7.cxx.o -MF graf2d/gpadv7/CMakeFiles/G__ROOTGpadv7.dir/G__ROOTGpadv7.cxx.o.d -o graf2d/gpadv7/CMakeFiles/G__ROOTGpadv7.dir/G__ROOTGpadv7.cxx.o -c /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-84d753c6ba5761703230a407cc87c9eb/build/graf2d/gpadv7/G__ROOTGpadv7.cxx
ninja: build stopped: subcommand failed.
error: Bad exit status from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/tmp/rpm-tmp.bIc3hk (%build)


RPM build errors:
line 37: It's not recommended to have unversioned Obsoletes: Obsoletes: lcg+root+6.31.01-84d753c6ba5761703230a407cc87c9eb
Macro expanded in comment on line 365: %{pkginstroot}/lib
</pre>

smuzaffar commented:
please test for CMSSW_14_0_ROOT6_X

cmsbuild commented:
-1

**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-fa6318/36783/summary.html
**COMMIT**: 9f208a4dc712accd3c4e0d70d583bf44151e8373
**CMSSW**: CMSSW_14_0_ROOT6_X_2024-01-09-2300/el8_amd64_gcc12
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/root/194/36783/install.sh` to create a dev area with all the needed externals and cmssw changes.


## External Build

I found compilation error when building: 

<pre>[8396/10519] /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin/g++  -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/io/sql/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/unix/../clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/unix/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/zstd/1.5.2-0530c990dd9ca27a8d966ab201ed2dfe/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/zstd/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/lz4/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/lzma/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/zip/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/thread/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/textinput/src -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/textinput/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/rint/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/meta/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/gui/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/cont/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clingutils/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clingutils/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clib/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/base/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/base/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/ginclude -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/io/io/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/io/io/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/tbb/v2021.9.0-5109e373398eaf79a8268d6a86a2f6e2/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/builtins -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/net/net/inc -D__ROOFIT_NOBANNER -fdiagnostics-color=always -std=c++17 -Wno-implicit-fallthrough -Wno-noexcept-type -pipe  -Wshadow -Wall -W -Woverloaded-virtual -fsigned-char -pthread -O3 -DNDEBUG -fPIC -std=c++17 -MD -MT io/sql/CMakeFiles/SQLIO.dir/src/TSQLStructure.cxx.o -MF io/sql/CMakeFiles/SQLIO.dir/src/TSQLStructure.cxx.o.d -o io/sql/CMakeFiles/SQLIO.dir/src/TSQLStructure.cxx.o -c /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/io/sql/src/TSQLStructure.cxx
[8397/10519] cd /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/core/imt && /cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02819/el8_amd64_gcc12/external/cmake/3.25.2-286587a5b04682914f36aa1a7526e333/bin/cmake -E env LD_LIBRARY_PATH=/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/tbb/v2021.9.0-5109e373398eaf79a8268d6a86a2f6e2/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xrootd/5.6.3-e577bbc9f9c02c6fbe1f70322e197ca2/lib64:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xrootd/5.6.3-e577bbc9f9c02c6fbe1f70322e197ca2/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/py3-numpy/1.24.3-6c4922f4e317e74c4b82416c41722d10/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/freetype/2.10.0-2f842c6203ecd373af1bd11574a751c3/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/pcre/8.43-37eb2e8b73bab83d6645ecfd5d73dcaa/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/libtiff/4.0.10-bdac8c539bf3892024554f734a79888f/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/dcap/2.47.12-5b22a716d41ee46990f5c71005a19f8d/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gsl/2.6-f316a321a173f181b66df52be79d894b/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/giflib/5.2.0-07dfc72586a7288f078c7a02c8b17956/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/lz4/1.9.2-e478fcd3e5e191d5bb4ade190474ad76/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/fftw3/3.3.8-5270bdf9998065b38f91e9aa053c73ec/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/libjpeg-turbo/2.0.2-cfe908f70a445c34076845aedc9ca824/lib64:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/libpng/1.6.37-7461873793d4834865bcbd73bf2bfcd6/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/py3-cython/0.29.35-b26088ec57407f91ea46c1fda0f722ef/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/OpenBLAS/0.3.15-15504a5c800a9ecbbc2befbb991bead5/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/hwloc/2.10.0-5498f051c9868674bf3304cef2a1fa56/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/libpciaccess/0.16-58aaa1607815fd64841e0504cabfc2b8/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/rocm/5.6.0-17877708168976351d54c3510af8ed71/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/py3-pip/23.1.1-4203b7dc85b43591db8a2c9fe2873971/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/py3-pip/23.1.1-4203b7dc85b43591db8a2c9fe2873971/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/cuda/12.2.1-bdf3fff69eaec65abe18a7569592cab6/lib64:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/zstd/1.5.2-0530c990dd9ca27a8d966ab201ed2dfe/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/libxml2/2.9.10-843ced916fa9921fbf8ea97048e64257/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/numactl/2.0.14-008b3e5466cf9216c5293916e495bd35/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/py3-setuptools/63.4.3-9776264c6f7d2bfa67dc48e0e798b861/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/curl/7.79.0-959a9188e043d67b2825f64cfeb54266/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/python3/3.9.14-4612d00f9f0430a19291545f1e47b4a4/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/python3/3.9.14-4612d00f9f0430a19291545f1e47b4a4/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/sqlite/3.36.0-fde8c1dbb8f282eaade54b13a9a9d4c6/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/libffi/3.4.2-0ff750af07ad87f96d240a17fa9b71fb/lib64:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gdbm/1.10-1f0ec10a142f21150eef9d32ba579779/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/db6/6.2.32-0b5a52d56859c2628499360eaabc790e/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/expat/2.4.8-b093687a482bf386f8f8c236c5b2efa2/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/bz2lib/1.0.6-d065ccd79984efc6d4660f410e4c81de/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/zlib/1.2.11-51072030b7f93c3ac6c4235f21e413cb/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/lib64:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/lib:/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/rpm/4.15.0-bfb05ca94c27dbcaf15444f2eb793405/lib:/.singularity.d/libs /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/core/rootcling_stage1/src/rootcling_stage1 -v2 -f G__Imt.cxx -cxxmodule -s /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/lib/libImt.so -m Core.pcm -m Thread.pcm -m MultiProc.pcm -excludePath /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01 -excludePath /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/ginclude -excludePath /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/externals -excludePath /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/builtins -compilerI/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/include/c++/12.3.1 -compilerI/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/include/c++/12.3.1/x86_64-redhat-linux-gnu -compilerI/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/include/c++/12.3.1/backward -compilerI/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/lib/gcc/x86_64-redhat-linux-gnu/12.3.1/include -compilerI/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/lib/gcc/x86_64-redhat-linux-gnu/12.3.1/include-fixed -compilerI/usr/local/include -compilerI/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/include -compilerI/usr/include -compilerI/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/lib/gcc/x86_64-redhat-linux-gnu/12.3.1/include -compilerI/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/lib/gcc/x86_64-redhat-linux-gnu/12.3.1/include-fixed -compilerI/usr/local/include -compilerI/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/include -compilerI/usr/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/imt/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/gui/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/cont/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clingutils/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clingutils/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clib/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/base/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/base/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/ginclude -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/tbb/v2021.9.0-5109e373398eaf79a8268d6a86a2f6e2/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/multiproc/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/meta/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/textinput/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/thread/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/zip/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/lzma/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/lz4/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/zstd/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/zstd/1.5.2-0530c990dd9ca27a8d966ab201ed2dfe/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/net/net/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/io/io/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/io/io/inc ROOT/TFuture.hxx ROOT/TTaskGroup.hxx ROOT/RTaskArena.hxx ROOT/RSlotStack.hxx ROOT/TExecutor.hxx ROOT/TThreadExecutor.hxx /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/imt/inc/LinkDef.h
[8398/10519] /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin/g++  -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/net/davix/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/unix/../clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/unix/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/zstd/1.5.2-0530c990dd9ca27a8d966ab201ed2dfe/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/zstd/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/lz4/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/lzma/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/zip/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/thread/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/textinput/src -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/textinput/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/rint/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/meta/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/gui/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/cont/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clingutils/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clingutils/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clib/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/base/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/base/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/ginclude -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/net/net/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/io/io/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/io/io/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/tbb/v2021.9.0-5109e373398eaf79a8268d6a86a2f6e2/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/builtins -isystem /cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02819/el8_amd64_gcc12/external/davix/0.8.1-bea8e5a05ee4ae615a84f825c78bc662/include/davix -D__ROOFIT_NOBANNER -fdiagnostics-color=always -std=c++17 -Wno-implicit-fallthrough -Wno-noexcept-type -pipe  -Wshadow -Wall -W -Woverloaded-virtual -fsigned-char -pthread -O3 -DNDEBUG -fPIC -Wno-deprecated-declarations -std=c++17 -MD -MT net/davix/CMakeFiles/RDAVIX.dir/src/TDavixFile.cxx.o -MF net/davix/CMakeFiles/RDAVIX.dir/src/TDavixFile.cxx.o.d -o net/davix/CMakeFiles/RDAVIX.dir/src/TDavixFile.cxx.o -c /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/net/davix/src/TDavixFile.cxx
[8399/10519] /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/gcc/12.3.1-40d504be6370b5a30e3947a6e575ca28/bin/g++  -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/graf2d/gpadv7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/zstd/1.5.2-0530c990dd9ca27a8d966ab201ed2dfe/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/xz/5.2.5-6f3f49b07db84e10c9be594a1176c114/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc12/external/tbb/v2021.9.0-5109e373398eaf79a8268d6a86a2f6e2/include -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/unix/../clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/unix/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/zstd/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/lz4/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/lzma/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/zip/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/thread/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/textinput/src -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/textinput/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/rint/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/meta/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/gui/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/foundation/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/cont/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clingutils/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clingutils/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clib/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/base/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/base/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/ginclude -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/io/io/v7/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/io/io/inc -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/core/clib/res -I/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/root-6.31.01/builtins -D__ROOFIT_NOBANNER -fdiagnostics-color=always -std=c++17 -Wno-implicit-fallthrough -Wno-noexcept-type -pipe  -Wshadow -Wall -W -Woverloaded-virtual -fsigned-char -pthread -O3 -DNDEBUG -fPIC -std=c++17 -MD -MT graf2d/gpadv7/CMakeFiles/G__ROOTGpadv7.dir/G__ROOTGpadv7.cxx.o -MF graf2d/gpadv7/CMakeFiles/G__ROOTGpadv7.dir/G__ROOTGpadv7.cxx.o.d -o graf2d/gpadv7/CMakeFiles/G__ROOTGpadv7.dir/G__ROOTGpadv7.cxx.o -c /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/BUILD/el8_amd64_gcc12/lcg/root/6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1/build/graf2d/gpadv7/G__ROOTGpadv7.cxx
ninja: build stopped: subcommand failed.
error: Bad exit status from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/tmp/rpm-tmp.FTHTmw (%build)


RPM build errors:
line 37: It's not recommended to have unversioned Obsoletes: Obsoletes: lcg+root+6.31.01-3d14dbdbecac20ecf3ce4771a8a442c1
Macro expanded in comment on line 365: %{pkginstroot}/lib
</pre>

smuzaffar commented:
please test with https://github.com/cms-sw/cmsdist/pull/8928 for CMSSW_14_0_ROOT6_X

cmsbuild commented:
+1

**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-fa6318/36790/summary.html
**COMMIT**: 9f208a4dc712accd3c4e0d70d583bf44151e8373
**CMSSW**: CMSSW_14_0_ROOT6_X_2024-01-09-2300/el8_amd64_gcc12
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/root/194/36790/install.sh` to create a dev area with all the needed externals and cmssw changes.



The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
 - @cms-sw cms-sw/cmsdist#8925

You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-fa6318/36790/git-recent-commits.json
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-fa6318/36790/git-merge-result


## Comparison Summary



**Summary**:
 - You potentially [added 106 lines](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_0_X_2024-01-09-2300+fa6318/60543/validateJR/logRootQA-events.log) to the logs
 - [Reco comparison results: 1010 differences found in the comparisons](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_0_X_2024-01-09-2300+fa6318/60543/validateJR.html)
 - DQMHistoTests: Total files compared: 48
 - DQMHistoTests: Total histograms compared: 3247277
 - DQMHistoTests: [Total failures: 7339](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_0_X_2024-01-09-2300+fa6318/60543/validateJR.html)
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 3239916
 - DQMHistoTests: Total skipped: 22
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 47 files compared)
 - Checked 200 log files, 161 edm output root files, 48 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_14_0_X_2024-01-09-2300+fa6318/60543/triggerResults): no differences found

agheata committed:
Fixes for TGeoTessellated and TGeoVGShape persistency. (#14327)

The previous implementation required TGeoTessellated shapes to be read attached to a TGeoManager. This removes the limitation, and allows also reading geometry files containing shapes converted to VecGeom corresponding solids. Fixes ##14283

cmsbuild commented:
Pull request #194 was updated.
